contracts/utils/VaultDeploymentHelper.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract VaultDeploymentHelper is Ownable {
+    /**
+     * @notice Emitted when a new vault is deployed
+     * @param vaultAddress the address of the new vault
+     */
+    event NewVault(address vaultAddress);
+
```

Comment:
1. maybe we can also maintain an **array** of all the vault addresses we have and just update it. can be fixed size array + private counter we maintain. 
2. maybe maintain mapping from vault address => enum type

Reply:
Added mapping enum type => array of all vaults with that type

Currently, the contract has been renamed to `VaultDeploymentEventEmitter`. Should we rename it to "AddressBook" or keep it as it is?

Reply:
Lets change to AddressBook since it serves that purpose as well - emitter can be secondary